src/lifetracker/LifeTracker.java

```diff
@@ -14,21 +15,32 @@
 import java.util.logging.LogManager;
 import java.util.logging.Logger;
 
-public class LifeTracker {
+import com.sun.j3d.utils.scenegraph.io.retained.Controller;
```

Comment:
Is this import necessary? It's an external library which doesn't seem to be used.